postgresql/deploy.nix

```
@@ -0,0 +1,223 @@
+let
+  region = "eu-central-1";
+  accessKeyId = "supabase-dev"; ## aws profile
+  sysbench = import ./sysbench.nix { stdenv = pkgs.stdenv;  fetchFromGitHub = pkgs.fetchFromGitHub ; autoreconfHook = pkgs.autoreconfHook ; pkg-config = pkgs.pkg-config ; libaio = pkgs.libaio ; postgresql = pkgs.postgresql_14; };
```

You can use [callPackage](https://nixos.org/guides/nix-pills/callpackage-design-pattern.html) to save some typing here:

```suggestion
  sysbench = pkgs.callPackage ./sysbench.nix { postgresql = pkgs.postgresql_14; };
```

---

I think `postgresql_14` is not the default from this `nixpkgs` version.. so leaving it there just in case.